bot/handlers/user/subscription/payments_subscription.py

```
@@ -0,0 +1,184 @@
+        price_rub,
+        stars_price,
+        currency_symbol_val,
+        current_lang,
+        i18n,
+        settings,
+        sale_mode="traffic" if traffic_mode else "subscription",
+    )
+
+    try:
+        await callback.message.edit_text(text_content, reply_markup=reply_markup)
+    except Exception as e_edit:
```

Ruff (0.15.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)